src/wallet/rpcwallet.cpp

```diff
@@ -290,14 +295,14 @@ UniValue setlabel(const JSONRPCRequest& request)
 
     if (request.fHelp || request.params.size() < 1 || request.params.size() > 2)
         throw std::runtime_error(
-            "setlabel \"address\" \"label\"\n"
+            "setlabel \"bitcoinaddress\" \"label\"\n"
```

Comment:
> Why rename the arguments/return values to bitcoinaddress everywhere? 

Agree it would be better to stick to `address`, but just as a historical note, this wasn't a "gratuitious" API break when it was originally written in 8571fee617aa22a43d73efa2560cd2ddf55c2478, because it preceded #11536, so the `setlabel` RPC was brand new.